src/api-doc/api-doc.yaml

```diff
@@ -0,0 +1,2869 @@
+  '/mentoring/v1/questionsSet/read/{QuestionSetId}':
+    get:
+      summary: Get question set
+      tags:
+        - Question Set APIs
+      description: "This API is associated with getting questions set on mentoring \n - Then
+        Endpoint for get question set `/mentoring/v1/questionsSet/read` \n - It is mandatory to
+        provide values for parameters marked with `required` \n - Mandatory parameter
+        cannot be empty or null"
+      parameters:
+        - name: X-auth-token
+          in: header
+          description: To make use of the API, you require X-auth-token. This is Available
+            in login API Response.
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: QuestionSetId
+          required: true
+          description: Please append a valid question set id in url
+          schema:
+            type: string
+          example: 629978caf0cf0901d7d74d12
+      responses:
+        '200':
+          description: Ok
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/questionset/readQuestionSetResponse200'
+        '400':
+          description: Bad Request
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/questionset/readQuestionSetResponse400'
+  '/mentoring/v1/feedback/forms/{SessionId}':
+    get:
+      summary: Get feedback form
+      tags:
+        - Feedback Form APIs
+      description: "This API is associated with getting feedback form in mentoring \n - Then
+        Endpoint for get feedback form `/mentoring/v1/feedback/forms` \n - It is mandatory to
+        provide values for parameters marked with `required` \n - Mandatory parameter
+        cannot be empty or null"
+      parameters:
+        - name: X-auth-token
+          in: header
+          description: To make use of the API, you require X-auth-token. This is Available
+            in login API Response.
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: SessionId
+          required: true
+          description: Please append a valid session id in url
+          schema:
+            type: string
+          example: 629978caf0cf0901d7d74d12
+      responses:
+        '200':
+          description: Ok
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/feedback/getFeedbackFromResponse200'
+        '400':
+          description: Bad Request
+          content:
+            application.json:
+              schema:
+                '$ref': '#/components/schemas/feedback/getFeedbackFormResponse400'
+  '/mentoring/v1/feedback/submit/{SessionId}':
+    post:
+      summary: Submit feedback
+      tags:
+        - Feedback Form APIs
+      description: "This API is associated with submit feedback form in mentoring \n - Then
```

Comment:
@ankitpws  not require ( in mentoring ) as its a mentoring api doc

Reply:
check other apis also

Reply:
fixed